lib/actions/addLinkToProjectAction.test.js

```diff
@@ -7,10 +7,10 @@ describe('Add Action', ()=> {
     it('@asyncResolves : should return an add object containing a response', async () => {
         const addLinkToProjectAction = new AddLinkToProjectAction();
 
-        const response = await addLinkToProjectAction.execute(['google', 'google.com'], {links: {}})
+        const response = await addLinkToProjectAction.execute(['google', 'google.com'], {})
 
-        const result = { label: 'google', link: 'google.com', project: { links: {} } }
-        expect(response).to.deep.equal(result);
+        const expectedResult = { label: 'google', link: 'google.com', project: {}}
+        expect(response).to.deep.equal(expectedResult);
```

Comment:
I would not expect (see what I did there) project to be asserted at all. We don’t yet know what we need in the `response`. Once we do, we should assert it explicitly. 

The error you will get with `deep.equal` is pretty hard to read *and* it’s too explicit.  By asking for exact match, the response instance cannot change for other tests.

Reply:
I understanddddd! ok ok i get it. I was wayy on another track for a sec hahaha.. I get what you mean. We wouldnt need this file then until we know what will be in the response.

Reply:
You need this file. You don't need to `deepEqual`:

```
expect(response.label).to.equal('google');
expect(response.value).to.equal('google.com');
```

Now you are directly worrying about the scope you care for. Nothing more. Nothing less.